### ISSUE TITLE
anki not appearing in list of apps to allow through firewall

### ISSUE THREAD

**Issue by kaleo333** [TheDabblingDuck/ankimote#34]
I've used this addon for a few different computers but on the newest computer, anki doesn't show up as an option of a program to manually allow through the firewall for private and public. I'm not sure if I'm missing some obvious way to make it show up. Thanks!

**Comment by TheDabblingDuck** [comment]
Hmm can't say I'm sure about that. Is Ankimote not working because of the firewall?

**Comment by kaleo333** [comment]
yes it's doing the thing where the page on the remote phone just never loads
